rudder-webapp/debian/postinst

```diff
@@ -88,6 +81,13 @@ case "$1" in
     fi
   fi
 
+  # Run any upgrades
+  # Note this must happen *before* creating the technique store, as it was moved in version 2.3.2
```

Comment:
Comments that say the contrary of the code are always a pleasure, we should keep them 🤦‍